packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselActionCards.stories.tsx

```diff
@@ -164,6 +164,10 @@ const ActionCard: React.FC<Post & { index: number }> = props => {
   );
 };
 
+const Announcement: CarouselAnnouncerFunction = (index: number, totalSlides: number, slideGroupList: number[][]) => {
```

Comment:
```suggestion
const getAnnouncement: CarouselAnnouncerFunction = (index: number, totalSlides: number, slideGroupList: number[][]) => {
```

Here and in other places: let's not use PascalCase as it stays for React components